akvo/rsr/models/user.py

```diff
@@ -376,16 +376,14 @@ def approved_employments(self):
         return self.employers.all().exclude(is_approved=False)
 
     def can_create_project(self):
-        """
-        Check to see if the user can create a project, or is a superuser.
+        """Check to see if the user can create a project."""
 
-        :return: Boolean to indicate whether the user has a reportable organisation
-        """
-        if self.is_superuser or self.is_admin:
+        if self.is_superuser:
```

Comment:
Let's keep god-mode. Not sure about self.is_admin, I think this is Akvo staff only and if so it might apply to them too?